lib/index.d.ts

```diff
@@ -6,22 +6,27 @@ export class Boom<Data = any> extends Error {
     /**
```

Comment:
Since we are introducing such a breaking change, would it be type safer to use `Data = unknown` instead of `any` ?

Reply:
That might make sense to require casting on access to `data` on plain Boom objects. It should go in a separate PR though.

Reply:
I changed it to unknown.